demo/src/Fracture.gd

```diff
@@ -181,5 +181,5 @@ func _on_FracturesSlider_value_changed(value: float) -> void:
 
 
 func _on_MinAreaSlider_value_changed(value):
-	_min_area_label.text = "Min Area: %d" % value
```

Comment:
I think this should still be Area instead of Area3D?

Reply:
Sure! I'm don't really know why it was changed.